sim/global-types.ts

```diff
@@ -1253,6 +1253,7 @@ interface FormatData extends EventMethods {
 	column?: number;
 }
 
+type FormatList = FormatData | {section: string, column?: number};
```

Comment:
Hm, in hindsight, I'm not sure `FormatList` is a great name for this type. Maybe if it also contained `[]`?

Reply:
```suggestion
type FormatList = (FormatData | {section: string, column?: number})[];
```

Like this?

Reply:
all instances use it as a list so I see -1 reasons to not do this.